tests/copyoffload/test_copyoffload_migration.py

```
@@ -405,14 +414,63 @@ def test_check_vms(
+        vm_namespace = prepared_plan["_vm_target_namespace"]
+
+        vm = VirtualMachine(
+            client=ocp_admin_client,
+            name=vm_name,
+            namespace=vm_namespace,
+        )
+
+        source_vm = source_provider.vm_dict(name=vm_name)
+
+        if not vm.ready:
+            LOGGER.info(f"Starting migrated VM {vm_name} for data integrity check")
```

Ruff (0.15.9)

[warning] Logging statement uses f-string

(G004)